src/components/RadioGroup/RadioGroups.jsx

```diff
@@ -0,0 +1,37 @@
+// eslint-disable-next-line no-unused-vars
+import React, { Component } from 'react'
+// eslint-disable-next-line no-unused-vars
+import { RadioGroup, Radio } from 'react-radio-group';
+
+class RadioGroups extends Component {
+render() {
+    console.log(this.props);
```

Comment:
Remove console.log

Reply:
done i'll push the updated code and will let  you know.